### ISSUE TITLE
Aromatic L-Amino Acid Decarboxylase enzyme low in plasma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10262]
**Preferred term label:**
Aromatic L-Amino Acid Decarboxylase enzyme low in plasma

**Synonyms**
AADC

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Aromatic L-amino acid decarboxylase deficiency
https://www.ncbi.nlm.nih.gov/books/NBK595821/ "measurement of ... plasma AADC enzyme activity is recommended if the pathogenic variants in the family are not known"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000037
- label: Reduced circulating aromatic L-amino acid decarboxylase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:17240182
- def: The activity of aromatic L-amino acid decarboxylase (also known as DOPA decarboxylase, tryptophan decarboxylase, and 5-hydroxytryptophan decarboxylase) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10262
- synonyms: Reduced circulating DOPA decarboxylase, activity(NONE)|Reduced circulating 5-hydroxytryptophan decarboxylase activity(NONE)
